<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDeleteDialog</class>
 <widget class="QDialog" name="MyDeleteDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除事件框</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="way_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>从不重复</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>每周</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>每月</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>每x天</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>160</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="OK_button">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>从</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="from_date"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>到</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="to_date"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
